src/main/java/ch/interlis/iox_j/validator/Validator.java

```diff
@@ -661,26 +661,28 @@ private void collectAdditionalConstraints(DataModel model){
 	private void validateAllAreas() {
 		setCurrentMainObj(null);
 		for(AttributeDef attr:areaAttrs.keySet()){
-			errs.addEvent(errFact.logInfoMsg(rsrc.getString("validateAllAreas.validateAREA"), getScopedName(attr)));
 			ItfAreaPolygon2Linetable allLines=areaAttrs.get(attr);
```

Comment:
statt dem public field in ItfAreaPolygon2Linetable ein Validator.areaAttrsStatus Map<AttributeDef,Boolean>